Patches/ODZ Stalingrad - Uniforms/ThingDefs_Apparel.xml

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="utf-8" ?>
+
+<Patch>
+    <Operation Class="PatchOperationFindMod">
+        <mods>
+            <li>[1.3] Stalingrad - Uniforms</li>
+        </mods>
+        <match Class="PatchOperationSequence">
+            <operations>
+
+                <!-- ======= Webbings ======= -->
+
+                <li Class="PatchOperationAdd">
+                    <xpath>Defs/ThingDef[ defName="ODZ_WWII_GermanRiflemanKit" or defName="ODZ_WWII_GermanAssaultKit" or defName="ODZ_WWII_GermanMGKit" or defName="ODZ_WWII_GermanPlatoonLeaderKit" or defName="ODZ_WWII_SovietRiflemanKit" or defName="ODZ_WWII_SovietAssaultKit" or defName="ODZ_WWII_SovietMGKit" or defName="ODZ_WWII_SovietPlatoonLeaderKit"]/statBases</xpath>
+                    <value>
+                        <Bulk>5</Bulk>
```

Comment:
Needs a wornBulk, as well. The CE TacVest has 5 wornBulk, so around that may be appropriate for this gear.